00-basics/002-javascript/Kristina/Learning 004.js

```diff
@@ -0,0 +1,158 @@
+function welcomeToBooleans() {
+    return true;
+};
+console.log(welcomeToBooleans);
+
+function test (myCondition) {
+    if (myCondition) {
+        return "It was true";
+    }
+    return "It was false";
+};
+console.log(test(true));
+console.log(test(false));
+
+function trueOrFalse(wasThatTrue) {
+    if(wasThatTrue){
+        return "Yes, that was true";
+    }
+    return "No, that was false"
+};
+console.log(trueOrFalse(false));
+
+function testEqual(val) {
+    if (val==12) {
+        return "Equal";
+    }
+    return "Not Equal";
+};
+console.log(testEqual(10));
+
+function testStrict(val) {
+    if (val===7) {
+        return "Equal";
+    }
+    return "Not Equal";
+}
+console.log(testStrict(10));
```

Comment:
to understand difference between strict and non-strict equality you should also try passing strings

Reply:
Had that in examples multiple times, found it easy so didn't recreate it. But thanks for the tip!